### PR TITLE
Only consider second file extension if primary ends with '.gz'

### PR DESCRIPTION
**Pull request by akarve**
Customer stacks with lot of *.blah.png, *.boo.png, *.baz.png extensions actually wanted all of those file stats to roll up under .png in the overview. Now we only consider the secondary extension if the primary is .gz.

### CODE REVIEW

File: lambdas/access_counts/index.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def sql_escape(s):
             eventname,
             bucket,
             lower(CASE
-                WHEN cardinality(parts) > 2 THEN concat(element_at(parts, -2), '.', element_at(parts, -1))
+                WHEN cardinality(parts) > 2 AND lower(element_at(parts, -1)) = 'gz' THEN concat(element_at(parts, -2), '.', element_at(parts, -1))
                 WHEN cardinality(parts) = 2 THEN element_at(parts, -1)
```
Comment (akarve) [inline lambdas/access_counts/index.py:214]:
```suggestion
                WHEN cardinality(parts) >= 2 THEN element_at(parts, -1)
```

File: lambdas/access_counts/index.py [hunk 2]
```diff
@@ -210,8 +210,9 @@ def sql_escape(s):
             eventname,
             bucket,
             lower(CASE
-                WHEN cardinality(parts) > 2 THEN concat(element_at(parts, -2), '.', element_at(parts, -1))
-                WHEN cardinality(parts) = 2 THEN element_at(parts, -1)
+                WHEN cardinality(parts) > 2 AND lower(element_at(parts, -1)) = 'gz'
+                    THEN concat(element_at(parts, -2), '.', element_at(parts, -1))
+                WHEN cardinality(parts) >= 2 THEN element_at(parts, -1)
```
Comment (dimaryaz) [inline lambdas/access_counts/index.py:215]: Why did you change it to `>=` here?

Comment (akarve) [reply lambdas/access_counts/index.py:215]: So that when `bucket/folder/foo.baz.png` flunks the line above, it still turns into  png?
